minerl/herobraine/hero/handlers/translation.py

```diff
@@ -62,6 +63,8 @@ def __init__(self,
         self.hero_keys = hero_keys
         self.univ_keys = univ_keys
         self.default_if_missing = default_if_missing
+        self.ignore_missing = ignore_missing
```

Comment:
Could you add a docstring for `ignore_missing` here?

Reply:
got that